src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -32,6 +41,35 @@ let begin;
 let pc;
 let candidates;
 
+const allServersKey = 'servers';
+
+function writeServersToLocalStorage() {
+  const serversSelect = document.querySelector('select#servers');
+  const allServers = JSON.stringify(Object.values(serversSelect.options).map(o => JSON.parse(o.value)));
+  window.localStorage.setItem(allServersKey, allServers);
+}
+
+function readServersFromLocalStorage() {
+  document.querySelectorAll('select#servers option').forEach(option => option.remove());
+  const serversSelect = document.querySelector('select#servers');
+  const storedServers = window.localStorage.getItem(allServersKey);
+
+  if (storedServers === null || storedServers === '') {
+    const o = document.createElement('option');
+    o.value = '{"urls":["stun:stun.l.google.com:19302"]}';
+    o.text = 'stun:stun.l.google.com:19302';
+    serversSelect.add(o);
+  } else {
+    JSON.parse(storedServers).forEach((server, key) => {
+      const o = document.createElement('option');
+      o.value = JSON.stringify(server);
+      o.text = server.urls[0];
+      o.ondblclick = selectServer;
```

Comment:
Not in this CL (it was there already), but how discoverable is a dobule-click interface?
What's it supposed to do?

Reply:
Agree. It seems it was added to quickly add new servers. I'll remove this for now as it is just confusing.